src/main/java/org/qualityannotate/quality/sonarqube/SonarqubeQualityTool.java

```diff
@@ -22,50 +21,58 @@ public class SonarqubeQualityTool implements QualityTool {
     public SonarqubeQualityTool(SonarqubeConfig config) {
         this.config = config;
         client = QuarkusRestClientBuilder.newBuilder()
-                                         .baseUri(URI.create(config.url()))
-                                         .register(new BasicAuthRequestFilter("", config.token()))
-                                         .build(SonarqubeApiClient.class);
+                .baseUri(URI.create(config.url()))
+                .register(new BasicAuthRequestFilter(config.token()))
+                .build(SonarqubeApiClient.class);
     }
 
     List<Issue> getIssues() {
-        IssueSearch issuesSearch =
-                client.getIssuesSearch(config.project(), config.pullRequest(), null, null, null, null, null, null, null,
-                        null, null);
+        IssueSearch issuesSearch = client.getIssuesSearch(config.project(), config.pullRequest(), null);
         // TODO issueSearch.facets can be used for globalMetrics - but maybe doesn't matter
         List<Issue> issues = new ArrayList<>();
         for (SqIssue sqIssue : issuesSearch.issues()) {
```

Comment:
ℹ️ MAINTAINABILITY: Remove this use of "severity"; it is deprecated. [details](https://sonarcloud.io/organizations/quyt/rules?open=java:S1874&rule_key=java:S1874)